WLC/tests.py

```diff
@@ -0,0 +1,11 @@
+from unittest import TestCase
```

Comment:
You don't really need to use `unittest` if you are running the tests with `pytest`.

Reply:
Fixed